scenariolib/visit.go

```diff
@@ -324,6 +318,14 @@ func (v *Visit) sendClickEvent(rank int, quickview bool, customData map[string]i
 		return errors.New("Cannot convert syssource to string")
 	}
 
+	event.QueryPipeline = v.LastResponse.Pipeline
+	event.Username = v.Username
+	event.Anonymous = v.Anonymous
+	event.Language = v.Language
+	event.OriginLevel1 = v.OriginLevel1
+	event.OriginLevel2 = v.OriginLevel2
+
+
```

Comment:
Please keep all the fixed assignations on top of the conditional ones for clarity